src/net.cpp

```diff
@@ -2616,7 +2598,6 @@ int CConnman::GetBestHeight() const
 }
 
 unsigned int CConnman::GetReceiveFloodSize() const { return nReceiveFloodSize; }
-unsigned int CConnman::GetSendBufferSize() const{ return nSendBufferMaxSize; }
```

Comment:
CConnman is recent code and part of an ongoing effort by @theuni, please discuss with him before removing things.